src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,673 @@
+:bug-report/inspector-page-copy-notif "Copiado para a área de transferência!"
+:bug-report/inspector-page-btn-create-issue "Criar uma issue"
+:bug-report/inspector-page-desc-clipboard "Aqui estão os dados lidos da área de transferência."
+:bug-report/inspector-page-desc-copy "Se for seguro compartilhar, clique no botão de cópia."
+:bug-report/inspector-page-desc-create-issue "Agora você pode relatar o resultado colado na sua área de transferência. Cole o resultado na seção 'Contexto Adicional' e indique de onde você copiou o conteúdo original. Obrigado!"
+:help/title-usage "Uso"
+:help/title-community "Comunidade"
+:help/title-development "Desenvolvimento"
+:help/title-about "Sobre"
+:help/title-terms "Termos"
+:help/start "Começando"
+:help/about "Sobre o Logseq"
+:help/roadmap "Roteiro"
+:help/bug "Relatório de bug"
+:help/feature "Solicitação de recurso"
+:help/changelog "Registro de alterações"
+:help/blog "Blog do Logseq"
+:help/docs "Documentação"
+:help/privacy "Política de privacidade"
+:help/terms "Termos"
+:help/forum-community "Comunidade do Fórum"
+:help/awesome-logseq "Incrível Logseq"
+:help/shortcuts "Atalhos de teclado"
  :help/shortcuts-triggers "Gatilhos"
  :help/shortcut "Atalho"
- :help/slash-autocomplete "Auto-completar com a barra"
- :help/block-content-autocomplete "Auto-completar conteúdo de bloco (Src, Quote, Query, etc)"
- :help/reference-autocomplete "Auto-completar referência de páginas"
+ :help/slash-autocomplete "Barra de preenchimento automático"
+ :help/block-content-autocomplete "Bloquear preenchimento automático de conteúdo"
+ :help/reference-autocomplete "Preenchimento automático de referência de página"
  :help/block-reference "Referência de bloco"
+ :help/open-link-in-sidebar "Abrir link na barra lateral"
  :more "Mais"
- :search/result-for "Resultado da pesquisa para "
- :help/context-menu "Menu contextual"
+ :search/result-for "Resultado da pesquisa por"
+ :search/items "itens"
+ :search/page-names "Pesquisar nomes de páginas"
+ :search/recent "Pesquisa recente:"
+ :search/blocks-in-page "Blocos de pesquisa na página:"
+ :search/command-palette-tip-1 "Dica: "
+ :search/command-palette-tip-2 " para abrir a paleta de comandos"
+ :search/cache-outdated "O cache está desatualizado. Clique no botão 'Reindexar' no menu suspenso do gráfico."
+ :search-item/page "Página"
+ :search-item/file "Arquivo"
+ :search-item/block "Bloco"
+ :search-item/no-result "Nenhum resultado correspondente"
+ :help/context-menu "Bloquear menu de contexto"
  :help/markdown-syntax "Sintaxe de Markdown"
- :help/org-mode-syntax "Sintaxe de modo Org"
+ :help/org-mode-syntax "Sintaxe de Org mode"
  :bold "Negrito"
  :italics "Itálico"
- :highlight "Realçado"
- :strikethrough "Rasurado"
+ :highlight "Destaque"
+ :strikethrough "Tachado"
  :code "Código"
+ :untitled "Sem título"
  :right-side-bar/help "Ajuda"
- :right-side-bar/switch-theme "Temas"
+ :right-side-bar/switch-theme "Modos temáticos"
```

Comment:
I think that in Portuguese it sounds better "Temas" itself. What is your opinion? Can we change it back?

Reply:
I made the correction.